docs/reference/HailExpressionLanguage.md

```diff
@@ -87,6 +87,8 @@ Several Hail commands provide the ability to perform a broad array of computatio
  - Numeric Array Operations:
      - min: `arr.min` -- valid only for numeric arrays, returns the minimum value
      - max: `arr.max` -- valid only for numeric arrays, returns the minimum value
+     - median: `arr.median` -- valid only for numeric arrays, returns the median value
+     - mean: `arr.mean` -- valid only for numeric arrays, returns the mean as a `Double`
```

Comment:
Awesome!  I can't believe I didn't check breeze for median.

As always, only question is how we handle missing.  These will both treat missing as 0.0.  Is that what we want?  Might be OK for mean, but seems counter-intuitive for median.  Either way, can you clarify how missing is handled in the doc?